LICENSE

```diff
@@ -235,3 +235,12 @@ The following components are provided under the MIT license. See project link fo
 The text of each license is also included at licenses/LICENSE-[project].txt.
 
     logrus 1.7.0: https://github.com/sirupsen/logrus MIT
+
+========================================================================
+BSD licenses
+========================================================================
+
+The following components are provided under a BSD license. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+
+   third-party files from golang/go: https://github.com/golang/go  BSD
```

Comment:
```suggestion
   third-party/template files from golang/go: https://github.com/golang/go/tree/master/src/text/template  BSD
```

Reply:
How about the `third-party/internal` files ? They are also from golang/go